transport/BaseUdpEndpoint.cpp

```diff
@@ -66,8 +39,8 @@ BaseUdpEndpoint::BaseUdpEndpoint(const char* name,
     : _state(Endpoint::CLOSED),
       _name(name),
       _localPort(localPort),
-      _receiveJobs(jobManager, 16),
-      _sendJobs(jobManager, 16),
+      _receiveJobs(jobManager, maxSessionCount),
```

Comment:
A small queue here is fine for receive jobs as they are like 2, but the queue is also used for unreg jobs and that can fill the queue causing receive to halt forever.